### ISSUE TITLE
Where can i get the images (training and test) data

### ISSUE THREAD

**Issue by shahidammer** [rezafuad/vehicle-color-recognition#4]

**Comment by rezafuad** [comment]
http://cloud.eic.hust.edu.cn:8071/~pchen/project.html

**Comment by shahidammer** [comment]
Thanks :+1:

**Comment by shahidammer** [comment]
Is this the same data which you've mentioned in the paper itself? If not, where can i get that dataset? I am trying to replicate this on keras.

**Comment by rezafuad** [comment]
Yes, it same. You can use the split configuration provided in the github repo.

**Comment by shahidammer** [comment]
Great. Thanks alot

**Comment by AqsaHassan** [comment]
unable to load this page to download dataset.. Could anyone please share this dataset?
